setup/sql/5_4_hotfixes.php

```diff
@@ -1248,5 +1248,38 @@
     $ilDB->execute($updateStatement, [$row['certificate_content'], $row['id']]);
 }
 ?>
+<#83>
+<?php
+$checked_files = [];
+$defaulf_bg_img_path = '/certificates/default/background.jpg';
+$checked_files[$defaulf_bg_img_path] = file_exists(CLIENT_DATA_DIR . $defaulf_bg_img_path);
+
+$res = $ilDB->query('SELECT id, certificate_content_bu FROM il_cert_user_cert WHERE background_image_path = "[BACKGROUND_IMAGE]"');
```

Comment:
Did you have any proof/evidence that `background_image_path` contains the string `[BACKGROUND_IMAGE]` on a ILIAS 5.4.x installation without any additional certificate related plugins/patches/bulk migration jobs?

Reply:
Please use `$ilDB->quote('[BACKGROUND_IMAGE]', 'text')` instead.

Reply:
We are currently evaluating when this happened. For the moment, it seems that it was caused by a 5.4 update.
I tell you more, as far as we find something out.

Reply:
* This could not have been caused by any database hotfix. The only step using `[BACKGROUND_IMAGE]` (for the `certificate_content` field) is step 82.